Cargo.toml

```diff
@@ -25,19 +25,19 @@ path = "src/lib.rs"
 async-stream = { version = "^0.3" }
 async-trait = { version = "^0.1" }
 chrono = { version = "^0", optional = true }
-time = { version = "^0.2", optional = true }
+time = { version = "^0.3", optional = true }
 futures = { version = "^0.3" }
 futures-util = { version = "^0.3" }
 log = { version = "^0" }
 tracing = { version = "0.1", features = ["log"] }
 rust_decimal = { version = "^1", optional = true }
 sea-orm-macros = { version = "^0.8.0", path = "sea-orm-macros", optional = true }
-sea-query = { version = "^0.24.5", features = ["thread-safe"] }
+sea-query = { version = "^0.26.0", features = ["thread-safe"] }
```

Comment:
Maybe without pin minor version?

Reply:
Do you mean the patch version?
- `<major>.<minor>.<patch>`

Yeah, we can relax that